app/server/api/user_api.py

```diff
@@ -137,7 +137,7 @@ def post(self, user_id):
 
         return make_response(jsonify(response_object)), response_code
 
-    @requires_auth(allowed_roles={'ADMIN': 'subadmin'})
+    @requires_auth(allowed_roles={'ADMIN': 'admin'})
```

Comment:
this feels more reasonable. subadmin's should only be able to enrol users -- not edit user data